scripts/release.sh

```diff
@@ -342,3 +341,14 @@ function branch_release() {
       ${commitish} \
       ${TAG}
 }
+
+function branch_release() {
+    local tags="$(git tag | cut -d 'v' -f2 | cut -d '.' -f1-2 | sort | uniq )"
```

Comment:
You should not release on tags, as they can be different on a local repo. Furthermore, this will fail on Prow job as it doesn't fetch all remotes.

Reply:
So I can replace it with: git ls-remote --tags origin? What should work!